dpl-docs/views/ddox.inc.module-tree.dt

```diff
@@ -1,11 +1,5 @@
 - import ddox.entities;
 
-- Module getPackageModule(Package pack)
-	- import std.algorithm : find;
-	- if (auto pp = cast(Package)pack.parent)
-		- return pp.lookup!Module(pack.name);
-	- return null;
-
```

Comment:
Why remove?

Reply:
Because it's defined in [`ddox.inc.utils.dt`](https://github.com/rejectedsoftware/ddox/blob/8ceb3815552765630e9fa0dece63f35837d416ee/views/ddox.inc.utils.dt#L19) which is now in this scope too.